src/Tickets/Commerce/Gateways/PayPal/REST/Order_Endpoint.php

```diff
@@ -178,7 +178,9 @@ public function handle_update_order( WP_REST_Request $request ) {
 			return new WP_Error( 'tec-tc-gateway-paypal-nonexistent-order-id', null, $order );
 		}
 
-		$payer_id = $request->get_param( 'payer_id' );
```

Comment:
We need to use `get_param` but to use you need to register the param; On `update_order_args`